validation/manifest_test.go

```diff
@@ -221,35 +233,35 @@ func Test_ValidateManifest(t *testing.T) {
 		{
 			name: "mismatch-group-count",
 			ok:   false,
-			mgroups: []*types.ManifestGroup{
+			mgroups: []manifest.Group{
 				{
 					Name: "foo",
-					Services: []*types.ManifestService{
+					Services: []manifest.Service{
 						{
 							Name: "svc1",
-							Unit: &types.ResourceUnit{
-								CPU:    5,
-								Memory: 20,
-								Disk:   5,
+							Unit: types.Unit{
+								CPU:     5,
+								Memory:  20,
+								Storage: 5,
 							},
 							Count: 3,
 						},
 					},
 				},
 			},
-			dgroups: []*types.DeploymentGroup{},
+			dgroups: []dtypes.Group{},
 		},
 	}
 
 	for _, test := range tests {
-		m := &types.Manifest{Groups: test.mgroups}
-		err := validation.ValidateManifestWithDeployment(m, test.dgroups)
-
-		if test.ok {
-			assert.NoError(t, err, test.name)
-		} else {
-			assert.Error(t, err, test.name)
-		}
+		m := manifest.Manifest(test.mgroups)
+		err := validation.ValidateManifestWithDeployment(&m, test.dgroups)
+		assert.NoError(t, err, test.name)
```

Comment:
Lets have errors cases too in the tests 
```suggestion
		if test.ok {
			assert.NoError(t, err, test.name)
		} else {
			assert.Error(t, err, test.name)
		}
```